Kitodo/src/main/java/org/kitodo/production/services/command/EditDataScript.java

```diff
@@ -56,16 +56,7 @@ public void process(LegacyMetsModsDigitalDocumentHelper metadataFile, Process pr
      * @return a list of metadata of a structural element selected from the list.
      */
     public Collection<Metadata> getMetadataCollection(List<IncludedStructuralElement> allIncludedStructuralElements) {
-        Collection<Metadata> metadataCollection = Collections.emptyList();
-        if (!allIncludedStructuralElements.isEmpty()) {
-            for (IncludedStructuralElement allIncludedStructuralElement : allIncludedStructuralElements) {
-                if (!allIncludedStructuralElement.getMetadata().isEmpty()) {
-                    metadataCollection = allIncludedStructuralElement.getMetadata();
-                    break;
-                }
-            }
-        }
-        return metadataCollection;
+        return allIncludedStructuralElements.get(0).getMetadata();
```

Comment:
Retrieving some element by index from a list without checking whether the list contains elements at all is never a good idea. Don't rely on surrounding methods to guarantee the given list has an element at index '0'!